metadatastore/mds.py

```diff
@@ -106,9 +106,13 @@ def _runstart_col(self):
     def _runstop_col(self):
         if self.__runstop_col is None:
             self.__runstop_col = self._db.get_collection('run_stop')
-
-            self.__runstop_col.create_index([('run_start', pymongo.DESCENDING),
-                                            ('uid', pymongo.DESCENDING)],
+            if self.version == 0:
+                self.__runstop_col.create_index('run_start_id',
+                                                unique=True)
+            else:
```

Comment:
When I used this pattern in the `_events_col`, you suggested that the `else` block should raise. I actually disagree; I think it's fine to assume that if the code is unchanged, v2 can assume it uses the same indices as v1. So as far as I'm concerned this is fine, but Past Tom objects. :- D